From 064d3bbd30e3dc8d7096ee267de66b6d0a56cd02 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Mon, 16 Oct 2017 22:50:26 +0300
Subject: [PATCH] arm: Check for __ELF__ instead of !__APPLE__, for using
 .arch/.fpu

For windows, when building with armasm, we already filtered these out
with gas-preprocessor.

By filtering them out already in the source, we can also build directly
with clang for windows (which also require wrapping the assembler in
gas-preprocessor for converting instructions to thumb form, but
gas-preprocessor doesn't and shouldn't filter out them in the clang
configuration).
---
 common/arm/asm.S | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/arm/asm.S b/common/arm/asm.S
index 619a509..37a4229 100644
--- a/common/arm/asm.S
+++ b/common/arm/asm.S
@@ -28,7 +28,7 @@
 
 .syntax unified
 
-#ifndef __APPLE__
+#ifdef __ELF__
 .arch armv7-a
 .fpu neon
 #endif
-- 
2.7.4

